libs/community/tests/integration_tests/chat_models/test_friendli.py

```diff
@@ -0,0 +1,114 @@
+"""Test Friendli chat API."""
+
+import pytest
+from langchain_core.messages.ai import AIMessage
+from langchain_core.messages.human import HumanMessage
+from langchain_core.outputs.generation import Generation
+from langchain_core.outputs.llm_result import LLMResult
+
+from langchain_community.chat_models.friendli import ChatFriendli
+
+
+@pytest.fixture
+def friendli_chat() -> ChatFriendli:
+    """Friendli LLM for chat."""
+    return ChatFriendli(temperature=0, max_tokens=10)
+
+
+@pytest.mark.scheduled
```

Comment:
@kooyunmo scheduled testing requires us to have a Friendli token set up in the repo secrets. happy to set that up if you'd like to send us a token, can send to bagatur@langchain.dev. but don't think we should block this PR on that so for now would recommend removing pytest.mark.scheduled

Reply:
@baskaryan Thanks for the quick review! I removed `pytest.mark.scheduled` from all integrated test cases I added as you suggested.